file-formats/clas/clas.json

```diff
@@ -1,135 +1,121 @@
-  "$id": "http://github.com/SAP/abap-file-formats/file-formats/clas/clas.json",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/clas/clas.json",
+  "title": "Class Properties",
+  "description": "Class properties",
   "type": "object",
   "properties": {
     "$schema": {
-      "type": "string",
       "title": "Schema",
-      "description": "Format version"
+      "description": "Format version",
+      "type": "string"
     },
     "header": {
-      "type": "object",
       "title": "Header",
+      "description": "header",
+      "type": "object",
       "properties": {
         "description": {
-          "type": "string",
           "title": "Description",
-          "maxLength": 60,
-          "description": "Description of a class"
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
         },
-        "masterLanguage": {
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
           "type": "string",
-          "minLength": 2,
-          "maxLength": 2,
-          "title": "Master Language",
-          "description": "Master language of the development object\nAll translatable text shall be maintained in this language"
+          "maxLength": 2
         },
         "abapLanguageVersion": {
+          "title": "ABAP Language Version (source code object)",
+          "description": "ABAP language version for source code objects",
           "type": "string",
           "enum": [
             "standard",
-            "standardUnicode",
             "keyUser",
             "cloudDevelopment"
           ],
           "enumDescriptions": [
             "Standard",
-            "Standard unicode",
-            "ABAP for key user tools",
-            "ABAP for Cloud Development"
-          ],
-          "title": "Abap Language Version",
-          "description": "ABAP language version"
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ]
         }
       },
       "additionalProperties": false,
       "required": [
         "description",
-        "masterLanguage",
+        "originalLanguage",
         "abapLanguageVersion"
       ]
     },
     "category": {
+      "title": "Class Category",
+      "description": "Class category",
       "type": "string",
       "enum": [
-        "GeneralObjectType",
-        "ExitClass",
-        "PersistentClass",
-        "FactoryForPersistentClass",
-        "StatusClassForPersistentClass",
-        "ViewClass",
-        "ProxyClassForRemoteInterface",
-        "RFCProxyClass",
-        "ExceptionClass",
-        "BusinessClass",
-        "BusinessClassInterfaceForStaticComponents",
-        "BusinessClassInterfaceForInstance-dependentComponents",
-        "BSPApplicationClass",
-        "BasisClassForBSPElementHandlers",
-        "WebDynproRuntimeObject",
```

Comment:
we mentioned webDynproRuntimeObject category  for interfaces, should it be for classes mentioned too?